Origin: author unknown, this was split out from a direct source edit.
Description: Add make rules to build documentation in plain text format.
             The "txt" target needs to be invoked explicitly (not
	     hooked up to "all") and needs lynx installed when used.
Forwarded: no

Index: iproute/doc/Makefile
===================================================================
--- iproute.orig/doc/Makefile	2010-08-05 12:33:36.000000000 +0200
+++ iproute/doc/Makefile	2010-08-05 12:51:42.000000000 +0200
@@ -14,6 +14,7 @@
 PAGESPERPAGE=2
 
 HTMLFILES=$(subst .sgml,.html,$(shell echo *.sgml))
+TXTFILES=$(subst .sgml,.txt,$(shell echo *.sgml))
 DVIFILES=$(subst .ps,.dvi,$(PSFILES))
 
 
@@ -25,6 +26,8 @@
 
 dvi: $(DVIFILES)
 
+txt: $(TXTFILES)
+
 print: $(PSFILES)
 	$(LPR) $(PSFILES)
 
@@ -47,9 +50,13 @@
 %.html: %.sgml
 	$(SGML2HTML) $<
 
+%.txt: %.html
+	lynx -nolist -dump $< > $@
+		
+
 install:
 	install -m 0644 $(shell echo *.tex) $(DESTDIR)$(DOCDIR)
 	install -m 0644 $(shell echo *.sgml) $(DESTDIR)$(DOCDIR)
 
 clean:
-	rm -f *.aux *.log *.toc $(PSFILES) $(DVIFILES) *.html
+	rm -f *.aux *.log *.toc $(PSFILES) $(DVIFILES) *.html $(TXTFILES)
